tests/e2e/deployment-test.sh

```diff
@@ -84,6 +83,26 @@ else
     gcloud pubsub subscriptions create "${PUBSUB_SUBSCRIPTION}" --topic="${PUBSUB_TOPIC}" --ack-deadline=120 --message-retention-duration=86400
 fi
 
+# Create Log Router Sink
+if [[ $(gcloud logging sinks  list --filter=name:"${LOG_ROUTER}" --format="value(name)") ]]; then
+    echo "Log Router [${LOG_ROUTER}] already exists, skipping"
+else
+  gcloud logging sinks create "${LOG_ROUTER}" "pubsub.googleapis.com/projects/${GCP_PROJECT_ID}/topics/${PUBSUB_TOPIC}" \
+    --log-filter='(resource.type="cloud_function" AND resource.labels.function_name="sample_app") OR resource.type="cloud_run_revision"' --description="Simple Sink for E2E tests"
```

Comment:
I think we don't want to include cloud_run_revision logs in this sink "OR resource.type="cloud_run_revision"?